pkg/cmd/defaults.go

```diff
@@ -1,14 +1,35 @@
 package cmd
 
 import (
+	"fmt"
 	"net/http"
 	"net/http/pprof"
 
+	"github.com/fatih/color"
 	"github.com/jzelinskie/cobrautil"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/rs/zerolog"
 )
 
+// ServeExample creates an example usage string with the provided program name.
+func ServeExample(programName string) string {
+	return fmt.Sprintf(`	%s:
+		%s serve --grpc-preshared-key "somerandomkeyhere"
```

Comment:
nit: maybe a an explicit argument index here? https://pkg.go.dev/fmt#hdr-Explicit_argument_indexes

Reply:
i had no idea this existed!